src/components/target/index.js

```diff
@@ -1,25 +1,58 @@
 import { useState } from 'react';
+import { Marker, useMapEvents, Popup } from 'react-leaflet';
 
 import { v4 as uuid } from 'uuid';
+import Swal from 'sweetalert2';
 
-import { Marker, useMapEvents, Popup } from 'react-leaflet';
-
+import { useDeleteTargetMutation } from 'services/target/target';
 import myIcon from '../map/icon';
+import endpoints from 'constants/endpoints';
+import useTranslation from 'hooks/useTranslation';
+
+import './styles.scss';
```

Comment:
Love that you start importing on an specific order as we discussed on the previous PR! Nice